sound/soc/intel/boards/sof_sdw_max98373.c

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: GPL-2.0
+// Copyright (c) 2020 Intel Corporation
+
+/*
+ *  sof_sdw_max98373 - Helpers to handle 2x MAX98373 codec devices
+ *  from generic machine driver
+ */
+
+#include <linux/device.h>
+#include <linux/errno.h>
+#include <sound/soc.h>
+#include <sound/soc-acpi.h>
+#include "sof_sdw_common.h"
+#include "../../codecs/max98373.h"
+
+static const struct snd_soc_dapm_widget mx8373_widgets[] = {
+	SND_SOC_DAPM_SPK("Speaker", NULL),
+};
+
+/*
+ * dapm routes for max98373 will be registered dynamically according
+ * to the number of max98373 used. The first two entries will be registered
+ * for one codec case, and the last two entries are also registered
+ * if two max98373s are used.
```

Comment:
I think this comment is copy-paste, don't seem to apply here where you only support the 2 amp case?

It'd be good to have a case where you only deal with a left config for example, or only right.

Reply:
Ack, shall correct it